docs-next/pages/apis/context.mdx

```diff
@@ -2,6 +2,139 @@ import { Markdown } from '../../components/Page';
 
 # Context API
 
-(coming soon)
+The `KeystoneContext` type represents an [Apollo Server context](https://www.apollographql.com/docs/apollo-server/data/resolvers/#the-context-argument) object.
```

Comment:
While technically correct, I don't feel like this is the right first sentence.

Can we think of a way to reword this paragraph to put emphasis on the KeystoneContext being this super useful thing we extend that's how you access most of the functionality in the system when you're writing extensions like access control, hooks, resolvers, etc... and it happens to be implemented as an Apollo server context object so it's available to you everywhere.

Or even if you don't invert the order, I think we could be clearer that the point is you have an Apollo Server context and we put a bunch of APIs on it for you; this is the docs for what it contains.

... or something

Reply:
Yep, let me take another stab at framing this.

Reply:
One thing I'm thinking about here is the fact that anyone who lands on this page will almost definitely be coming from one of our other pages (e.g. hooks, access control, etc) where we've said "hey, this function gets given a `KeystoneContext`, go checkout out the docs\<link\>". This isn't the place where we have to sell the general purpose usefulness of it. That should be covered by our user guides.

Reply:
Also, calling it an "Apollo Server context" isn't right, it's not an Apollo thing, it's a graphql-js thing.